cluster-provision/gocli/go.mod

```diff
@@ -21,3 +21,5 @@ require (
 )
 
 replace github.com/Sirupsen/logrus => github.com/sirupsen/logrus v1.4.1
+
+go 1.13
```

Comment:
Is this a leftover?

Reply:
will reprovision and check,
(also will ask Roman when he is back, but wont block on that)